pywbem/cim_operations.py

```diff
@@ -114,6 +139,42 @@
 _ILL_FORMED_UTF8_RE = re.compile(
     b'(\xED[\xA0-\xBF][\x80-\xBF])')    # U+D800...U+DFFF
 
+class EnumContext(object):
```

Comment:
I believe that this class is not needed at all. Its namespace attr is not needed. Its max_obj_count prevents specifying a different obj count on pull (depending on pull interface). maybe you want zero on open and some non-zero value on pull. the end of seq can as well be a boolean out parm. that leaves the op level enum context which is a string. thexout params can be represented as returned tuples in python, and the current approach needs to do that anyway. overall, i think not enough advantages to have the class.